man/systemd.unit.xml

```diff
@@ -1646,6 +1646,19 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionOSRelease=</varname></term>
+
+          <listitem><para>Verify that a specific <literal>key=value</literal> pair is set in the host's
+          <citerefentry><refentrytitle>os-release</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
+
+          <para>Other than exact matching with <literal>=</literal> and <literal>!=</literal>, relative comparisons
+          are supported for versioned parameters (e.g. <literal>VERSION_ID</literal>). The comparator can be one of
+          <literal>&lt;</literal>, <literal>&lt;=</literal>, <literal>=</literal>, <literal>!=</literal>,
```

Comment:
I would prefer `==` as it's not ambiguous compared to the `=` assignment type. Moreover, it's more common to see equality represented with `==` rather than `=`.

Reply:
Yeah, but we have ConditionKernelVersion with "=" already. So the ship has pretty much sailed.

Reply:
Yep starting from scratch `==` would probably have been nicer, but I'd rather have internal consistency between options in the same group now

Reply:
Could we fix this over there too?

Reply:
alright, now both `=` and `==` are allowed in both

Reply:
there are some programming languages that do equality comparisons with "==" (i.e. C) and others which do it wth "=" (shell + /bin/test, pascal). It's not clear to me which one is more "natural". I'd argue that being closer to shell is probably the better idea here.

Given how vague this is, and given that redundancy sucks and is confusing, and given that we already have established "=" for this, I'd rather stay with that, and not introduce ==

Reply:
Dropped again